src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -33,7 +33,7 @@
         </service>
 
         <service id="translation.loader.yml" class="Symfony\Component\Translation\Loader\YamlFileLoader" public="true">
-            <tag name="translation.loader" alias="yml" />
+            <tag name="translation.loader" alias="yaml" legacy-alias="yml" />
```

Comment:
hmm, is that another bugfix?

Reply:
yes. It allow to find `.yml` and `.yaml` files for translations. I let this in this PR because I think it is part of the command fix